index.html

```diff
@@ -91,22 +106,28 @@ <h3>I'm a Software Developer</h3>
     <section class="work-section">
       <div class="works-container">
         <article class="Recent-work">
-          <h2 class="works-title">My Recent Works</h2>
-          <hr>
-          <img class="feature-image" src="./images/background-2.svg" alt="feature image about works">
-          <h3 class="title-post">Multi-Post Stories</h3>
-          <p class="post-description">
-            A daily selection of privately personalized reads;
-            no accounts or sign-ups required. has been the industry's standard
-            dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
-          </p>
-          <ul class="tag-ul-1">
-            <li>CSS</li>
-            <li>Html</li>
-            <li>Bootstrap</li>
-            <li>Ruby</li>
-          </ul>
-          <button type="button" autofocus> See Project</button>
+          <div class="work-title-dv">
+            <h2 class="works-title">My Recent Works</h2>
+            <hr>
+          </div>
+          <div class="feature-area">
```

Comment:
You set up hover but it shouldn't be on that way. This part is little bit tricky and the design might be confusing on the first look. When you hover over the card, "See project" button needs to be clickable. Check the gif below. Kindly implement these card like on the gif. 👍
![Screen Recording 2023-01-26 at 10 18 53 PM](https://user-images.githubusercontent.com/25887253/214931510-152ec2dd-b6c5-4099-8cac-52ab261820fa.gif)

Reply:
Hello @huseyinbiyik, Thanks for your suggestion It's not included Microverse requirements. I include below one example repo link that has passed without including that hover effect.
`https://github.com/Santosh-Konappanavar/PortFolio/pull/4`
our requirements link-`https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md`